### PR TITLE
Adding documentation for ASTER-L1T open dataset

### PR DESCRIPTION
**Pull request by piyushrpt**
*Description of changes:*
- Adding yaml file for ASTER-L1T open dataset

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/aster-l1t.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  possible (systematic terrain, systematic, or precision) for scenes acquired at night or that
+  otherwise represent a reduced quality ground image (e.g., cloud cover).
+
+  Each AST_L1T granule is converted into three different COG files based on the sensor and spatial resolution,
+  VNIR at 15m, SWIR at 30m and TIR at 90m. The metadata required to transform the digital numbers (DN) to 
+  radiance and reflectance values are also incorporated as metadata in the TIFF files.
+  The filenaming convention and the organization of bands are described
+  [here](https://github.com/awslabs/open-data-docs/tree/main/docs/aster-l1t).
+
+Documentation: https://github.com/awslabs/open-data-docs/tree/main/docs/aster-l1t
```
Comment (jeffejefe) [inline datasets/aster-l1t.yaml:21]: Hi @piyushrpt it doesn't look like the documentation is live. Can you post an active link to this?

File: datasets/aster-l1t.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Contact: opendata@descarteslabs.com
+ManagedBy: "[Descartes Labs](https://descarteslabs.com/)"
+UpdateFrequency: Daily
+Collabs:
```
Comment (jeffejefe) [inline datasets/aster-l1t.yaml:25]: @xmlnovelist this is a dataset from Descartes Labs that is proposed for ASDI inclusion. Can you confirm that it's a candidate for you? For your background, https://lpdaac.usgs.gov/products/ast_l1tv003/. It's satellite imagery from the USGS so applicable.

File: datasets/aster-l1t.yaml (hunk 1 above)
Comment (piyushrpt) [reply datasets/aster-l1t.yaml:21]: There is a PR open on the open-data-docs repo here:
https://github.com/awslabs/open-data-docs/pull/22

The source can be found here:
https://github.com/descarteslabs/open-data-docs/blob/feat/aster-l1t/docs/aster-l1t/README.md

Comment (jeffejefe) [reply datasets/aster-l1t.yaml:21]: Great I just merged the docs PR so that link now works.

**Comment by jeffejefe** [comment]
Thanks @piyushrpt

File: datasets/aster-l1t.yaml (hunk 2 above)
Comment (jeffejefe) [reply datasets/aster-l1t.yaml:25]: closed this in favor of https://github.com/awslabs/open-data-registry/pull/1685

**Pull request merged**